pages/home.js

```diff
@@ -12,28 +12,23 @@ const colors = {
   'blue' : '#003da5'
 }
 
-const download = (store, actions) => {
-  if(store.update.Red.length == 0) {
-    fetch('/puller/status').then((resp) => resp.json())
-    .then((resp) => {
-      actions.setData(resp);
-    });
-    
+const capitalize = (string) => string.split('').slice(0,1).join('').toUpperCase()+string.split('').slice(1).join('').toLowerCase()
+
+module.exports = (store, actions, params) => {
+  // if we haven't started pulling data, do so now
+  if (store.pull === 'NOT_PULLED') {
+    actions.startPulling()
   }
-};
 
-module.exports = (store, actions) => {
-  download(store, actions);
+  const lines = (params.color ? [params.color] : ['green', 'orange', 'red', /* 'blue' */])
+    .map(color => html`<line-status line='${capitalize(color)}' color='${colors[color]}' />`)
```

Comment:
if `params.color` is defined, use that, otherwise, use the array to generate a list of lines